design-proposals/eim-pxe-with-managed-emf.md

```diff
@@ -0,0 +1,200 @@
+# Design Proposal: Support PXE-based provisioning with cloud-based EMF
+
+Author(s): Edge Infrastructure Manager and App orchestration teams.
+
+## Abstract
+
+The design on [EMT-S scale provisioning](./emts-scale-provisioning.md) proposes adding a DHCP/TFTP server to support
+PXE-based OS provisioning. It assumes that the EIM part of EMF orchestrator is deployed locally, on OXM premises.
+
+In some cases it is mandatory  for users to bootstrap edge nodes via legacy PXE, but the EMF is deployed in a remote
+cloud-based solution (either native k8s or a VM but not on on-prem).
+
+This design proposal addresses the needs of the users that cannot adopt the HTTPs boot because not supported on low-end
+budgeted device and when the USB boot does not provide a scalable solution for thousands of nodes.
+
+## Proposal
+
+Similarly to [EMT-S scale provisioning](./emts-scale-provisioning.md), the solution assumes there is the EMF OXM profile
+deployed in the local network. This can be deployed using several options:
+
+- on site directly on metal or a VM (via installation script provided by the EIM);
+- on an edge cluster deployed as kube-virt VM;
+
+The second solution is appealing for scenarios where the customer/user does not have additional computing to host
+EIM-local and it deploys directly as edge workload on the existing edge clusters. Of course this will require special
+configuration and possibly bind the VM on the host-network to allow DHCP/TFTP traffic flowing through the
+virtualization layers.
+
+Differently from the naive OXM profile, this solution will serve the `boot.ipxe` and the Micro-OS image of the remote
+orchestrator. Once the Micro-OS is booted, the remote EMF will take control and spawn the cloud based provisioning flow.
+
+### MVP Requirements
+
+The need for this solution is to overcome the limitations of lower budgeted devices that cannot boot directly using the
+cloud based provisioning based on HTTPs. There is still need to manage these nodes from day 0 from a central cloud.
+
+There is no need to manage the node as standalone, however as discussed later in the [Rationale](#rationale) section this
+could be an alternative solution together with the ability of importing standalone nodes into a remote orchestrator.
+
+## Rationale
+
+An alternative solution is to deploy only a small local piece of Edge Infrastructure Manager (called "EIM-local"
+hereinafter), given its small footprint it makes possible to deploy it using several solutions:
+
+- on site directly on metal or a VM via installation script;
```

Comment:
It can also be run as standalone Docker container on metal. I'd say VM is only an overhead here